src/androidTest/java/tests/integration/streaming/StreamingDisabledTest.java

```diff
@@ -87,8 +96,8 @@ public void streamingDisabled() throws IOException, InterruptedException {
         Assert.assertFalse(mIsStreamingConnected);
 
         // More than 1 hits means polling enabled
-        Assert.assertEquals(3,  mySegmentsHitsCountHit);
-        Assert.assertEquals(3, mSplitsHitsCountHit);
+        Assert.assertTrue( mySegmentsHitsCountHit > 1);
```

Comment:
Unfortunately the style is not consistent between these classes. I'll keep it as is for now to avoid a massive number of changes; can fix it in a following PR.